setup.py

```diff
@@ -0,0 +1,31 @@
+from os import path
+from setuptools import setup, find_packages
+
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+
+def _read_dependencies():
+    requirements_file = "requirements.txt"
+    with open(requirements_file) as fin:
+        return [line.strip() for line in fin if line]
+
+
+packages = find_packages()
+requirements = _read_dependencies()
+
+
+setup(
+    name="opentutor_classifier",
+    version="1.0.0",
```

Comment:
at some point, we have to fix this so that the version comes from the git tag. I'm not 100% sure how it works. I see this:

https://pypi.org/project/setuptools-scm/

...but we can fix easier issues first. Just something to remember to fix when we can

Reply:
Gotcha! Let's fix that later.